web/src/components/tasks/business-formation/PaymentSection.tsx

```diff
@@ -85,7 +85,7 @@ export const PaymentSection = (props: Props): ReactElement => {
     }
   };
 
-  if (!userData || state.tab !== 3) return <></>;
+  if (!userData || state.tab !== 4) return <></>;
```

Comment:
Is there a way we could make the tab numbers not hardcoded? Maybe an object that maps each thing to its page number, for example?